notebooks/pandas_08_reshaping_data.md

```diff
@@ -136,7 +136,7 @@ The usage of the tidy data representation has some important benefits when worki
 data_long.groupby("Treatment")["pH"].mean()  # switch to `WWTP`
 ```
 
-```{code-cell} ipython3
+```python
```

Comment:
(small nit) This looks a bit strange without text between the previous code block and this markdown code block (since they are formatted so differently)

Reply:
Good point. I converted to markdown in the first place to have no error. I now added some context and information before and after the markdown codeblock.